container/utils.go

```diff
@@ -6,20 +6,39 @@ package container
 import (
 	"os/exec"
 	"runtime"
+
+	"github.com/juju/errors"
+	"github.com/juju/juju/core/instance"
 )
 
-var RunningInContainer = func() bool {
-	if runtime.GOOS != "linux" {
-		return false
+func pathForExcutable(executableName string) (string, error) {
+	var ePath string
+	if path, err := exec.LookPath(executableName); err == nil {
+		ePath = path
+	} else if path, err := exec.LookPath("/usr/bin/" + executableName); err == nil {
```

Comment:
why the explicit `/usr/bin` look path call?

Reply:
yeah that looks redundant, I think it was something else when I first wrote this, don't know why I kept that, good catch 👍